env/build_jet_intel.env

```diff
@@ -40,3 +41,27 @@ export CMAKE_CXX_COMPILER=mpiicpc
 export CMAKE_Fortran_COMPILER=mpiifort
 export CMAKE_Platform=jet.intel
 
+export NCEPLIBS="/lfs4/BMC/wrfruc/gge/precompiled/NCEPLIBS/b_intel18.0.5.274_impi2018.4.274/install"
+export GSILIBS="/lfs4/BMC/wrfruc/gge/precompiled/GSILIBS/b_intel18.0.5.274_impi2018.4.274"
+export BACIO_LIB4=${NCEPLIBS}/lib/libbacio_4.a
+export  BUFR_LIBd=${GSILIBS}/lib/libbufr_v.a
+export GSIWRF_LIB=${GSILIBS}/lib/libWRFLIB.a
+export   CRTM_LIB=${NCEPLIBS}/lib/libcrtm.a
+export   CRTM_INC=${NCEPLIBS}/include
+export NEMSIO_LIB=${NCEPLIBS}/lib/libnemsio.a
+export NEMSIO_INC=${NCEPLIBS}/include
+export SFCIO_LIB4=${NCEPLIBS}/lib/libsfcio_4.a
+export SFCIO_INC4=${NCEPLIBS}/include_4
+export SIGIO_LIB4=${NCEPLIBS}/lib/libsigio_4.a
+export SIGIO_INC4=${NCEPLIBS}/include_4
+export    SP_LIBd=${NCEPLIBS}/lib/libsp_d.a
+export    SP_LIB4=${NCEPLIBS}/lib/libsp_4.a
+export W3EMC_LIBd=${NCEPLIBS}/lib/libw3emc_d.a
+export W3EMC_LIB4=${NCEPLIBS}/lib/libw3emc_4.a
+export W3EMC_INCd=${NCEPLIBS}/include_d
+export W3EMC_INC4=${NCEPLIBS}/include_4
+export W3NCO_LIBd=${NCEPLIBS}/lib/libw3nco_d.a
+export W3NCO_LIB4=${NCEPLIBS}/lib/libw3nco_4.a
+export    IP_LIBd=${NCEPLIBS}/lib/libip_d.a
+export    IP_LIB4=${NCEPLIBS}/lib/libip_4.a
+
```

Comment:
Those are best combined environmental setups I can do for now. I will check if those can be set in other ways in next PR when we really use the GSI analysis.

Reply:
These environment variables are already set by the modules, and have the potential to cause problems by setting them here when building the other components. You should either use the full environment from the  NCEPLIBS path you are setting here, or use the modules, but not both. This may cause issues for the non-cycled configurations in this branch. Please address this in this PR.

Reply:
Those environment variables will not impact the non-cycled configuration and I have use those environment variables in dev2 for many days. We need to move forward as long as it works for now and get back to improve the compiling system. I may go back to use separate compiling environment variables for GSI and model if I cannot solve parallel netcdf IO for ensemble files. Please merge this PR if there are no other critical issues.